content/doc/book/scaling/architecting-for-scale.adoc

```diff
@@ -92,7 +92,7 @@ with which builds are scheduled depends on the configuration given to each
 project. For example, some projects may be configured to "restrict where this
 project is run" which ties the project to a specific agent or set of labeled
 agents. Other projects which omit this configuration will select an agent from
-the available pool in Jenkins.
+the available pool in Jenkins.Learn more about Agents in link:doc/book/managing/nodes/#creating-agents/[Managing Nodes]
```

Comment:
We prefer sentence per line for easier reviewing of new content.  Also need the leading '/' so that the link jumps to the correct page.
```suggestion
the available pool in Jenkins.
Learn more about Agents in link:/doc/book/managing/nodes/#creating-agents/[Managing Nodes]
```